pavelib/assets.py

```diff
@@ -455,8 +472,9 @@ def update_assets(args):
     compile_coffeescript()
     call_task('pavelib.assets.compile_sass', options={'system': args.system, 'debug': args.debug})
 
+    collectstatic_debug_dict = {"debug": args.debug, "collect_log": args.collect_log_file}
```

Comment:
Hmm...I was mainly trying to simplify the `collect_assets` function. If you think it's easier to have two args, then that's fine...